src/qt/forms/sendcoinsentry.ui

```diff
@@ -18,7 +18,7 @@
   </property>
   <widget class="QFrame" name="SendCoins">
    <property name="toolTip">
-    <string>This is a normal payment.</string>
+    <string>The amount to send in the selected unit</string>
```

Comment:
Could be the `payAmount` tooltip, in line 168?

Reply:
Let's not keep drawing this out